storage/range.go

```diff
@@ -909,12 +909,20 @@ func (r *Range) maybeGossipFirstRange() error {
 	if !r.IsFirstRange() {
 		return nil
 	}
+
 	ctx := r.context()
+
+	// Gossip the cluster ID from all replicas of the first range.
+	log.Infof("gossiping first range from store %d, range %d", r.rm.StoreID(), r.Desc().RaftID)
```

Comment:
Do you think we should move the various and sundry log messages about gossiping into gossip? I think a node gossiping information should probably always be recorded in its logs, and that would make all of those events consistent.